test/components/board/Board.test.js

```diff
@@ -0,0 +1,11 @@
+import Enzyme from 'enzyme';
+import React from 'react';
+import {Board} from '../../../src/components/board/Board';
+import {Cell} from '../../../src/components/cell/Cell';
+import Adapter from 'enzyme-adapter-react-16';
+Enzyme.configure({adapter: new Adapter()});
+
+it('displays 9 cells for a 3x3 board', () => {
+  const wrapper = Enzyme.shallow(<Board updatedGrid={['', '', '', '', '', '', '', '', '']}/>);
```

Comment:
What if you no longer wanted a blank mark to be an empty string?  I'd user your `src/cellValue.js` here.